mautic/mautic
# fixes silent error on twig function call

a.b 4.4.6

Bug fix? (use the a.b branch)          | [x]
| Automated tests included?              | [ ] not sure what tests to write, I have this running on our live instance of mautic with the fix in place.

| Issue(s) addressed                     | Fixes # This fixes a silent error that is thrown in a twig function call. If the user is still running php 7.4 a silent error is thrown when a landing page is loaded. It will then try to inject grapesjs javascript files that it cannot find which causes the landing page load time to drastically increase.

 Description:

Mautic v4 still supports php 7.4 there should be no code that is specific to php8 as it will cause Breaking changes. This fix removes the php8 syntax to keep backwards compatibility for the users who have not upgrade to php8 from php 7.4

Credit goes to @paulrose who found where the error was actually thrown.

#### Steps to test this PR:
1. Load this PR with gitpod and ensure you are running php 7.4
2. create a landing page that uses a template (preferably build using grapejs)
3. visit the public url (incognito is best)
4. check chrome dev tools and ensure that there is no errors that it cannot load js files (grapesjs files)
